rte_mis.profile

```diff
@@ -15,3 +16,16 @@ function rte_mis_install_tasks_alter(array &$tasks) {
   // site settings.
   $tasks['install_configure_form']['function'] = SiteConfigureForm::class;
 }
+
+/**
+ * Implements hook_update_N().
+ */
+function rte_mis_update_10006() {
```

Comment:
I think this should be added in `rte_mis.install` file.

Cc @kunal-sin

Reply:
@manish-3108 Updated the PR. @kunal-sin sir, please review.